clients/web/src/containers/syndicated-article/syndicated-article.js

```diff
@@ -88,36 +82,18 @@ export function SyndicatedArticle({ queryParams = validParams, locale }) {
   // modify rendered elements when query params are passed in
   const { mobile_web_view, premium_user } = queryParams
   const isMobileWebView = mobile_web_view === 'true'
-  const isPremiumUser = premium_user === 'true' || isPremium === '1'
+  const isPremiumUser = premium_user === 'true' || (isPremium === '1' && 1 === 2)
   const allowAds = userStatus === 'pending' || isPremiumUser ? false : showAds
   const showCTA = userStatus !== 'valid'
-
-  // Initialize Mozilla Placards on the page
-  const featureState = useSelector((state) => state.features)
-  const showPlacards = featureFlagActive({ flag: 'moz.placards', featureState })
-
-  useEffect(() => {
-    if (!showPlacards) return
-
-    dispatch(
-      getPlacards([
-        {
-          placement: 'pocket_billboard',
-          count: 2
-        },
-        {
-          placement: 'pocket_skyscraper',
-          count: 2
-        }
-      ])
-    )
-  }, [dispatch, iabSubCategoryId, iabTopCategoryId, showPlacards])
+  const resolved = userStatus !== 'pending'
 
   const { asPath: urlPath } = router
   const targeting = {
     URL: urlPath,
     Category: iabTopCategory,
     SubCategory: iabSubCategory,
+    iabTopCategoryId,
+    iabSubCategoryId,
```

Comment:
These I think are what you are actually looking for no?

Reply:
Oh yes, I believe you're right but I hadn't double checked this yet. Looking at the mars openapi yaml, the value should be an array of things like `IAB1-5`, which definitely seems like an ID. 

(SDK is not passing these values through to MARS yet btw, we have an outstanding ticket to hook that up, I think it's pretty small but hasn't happened yet)